pkg/client/typed/dynamic/dynamic_util.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"k8s.io/kubernetes/pkg/api/meta"
 	metav1 "k8s.io/kubernetes/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/apis/meta/v1/unstructured"
```

Comment:
I wouldn't mind importing this as `unstructuredv1`, making the difference to `runtime.Unstructured` clearer.